### PR TITLE
Feature/bootstrap

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
This can be reviewed. I'm going to setup Travis CI for this project as part of this PR but the config for that is minimal.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+node_js:
+  - '4'
+  - '5'
+  - '6'
+
+install:
+  - npm install
+
+script:
+  - npm test
```
Comment (rmartz) [inline .travis.yml:16]: Does it affect anything if we do `npm test` (Like here) or `npm run test` (Like in the README)? They seem to work the same

Comment (CloudNiner) [inline .travis.yml:16]: Nope. It's just a shortcut.

**Comment by CloudNiner** [comment]
Ready for final review

**Comment by rmartz** [comment]
:+1:

**Pull request closed without merging**